Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -74,6 +74,7 @@
   private final boolean myRemoteDebug;
   private final boolean myEntryPointInLibFolder;
   private final int myTimeout;
+  private final VirtualFile myCurrentWorkingDirectory;
```

Comment:
Cool, I updated this to `@Nullable` (and adjusted one use of myCurrentWorkingDirectory).